ush/config.sh.RRFS_CONUS_13km

```diff
@@ -25,6 +25,40 @@ DO_SOIL_ADJUST="TRUE"
 DO_RADDA="FALSE"
 DO_BUFRSND="FALSE"
 
+l_both_fv3sar_gfs_ens=.false. #if true, ensemble size is increased with GDAS ensemble
```

Comment:
This part is added in three configuration files: RRFS_CONUS_13km, RRFS_CONUS_3km, RRFS_CONUS_3km.2022HWT. If they are same or mostly same, can make a separate file just like
set_rrfs_config_general.sh and only include special setups in configure. You can also setup grid type related parameters in that file. Please let me know if you need more details on how to simplify this setups.